server/neptune/workflows/deploy_test.go

```diff
@@ -59,6 +59,9 @@ func TestDeployWorkflow(t *testing.T) {
 
 	// we should have output for 2 different jobs
 	assert.Len(t, s.streamCloser.CapturedJobOutput, 2)
+
+	// we should emit 2 IN_PROGRESS and SUCCESS events
```

Comment:
I'm unclear why 2 events need to be sent for in progress.  Is that how it is currently?

If not, should this be discussed? It might be fine in our case but I'm not sure generally we should make assumptions that downstream services are resilient to duplicates

Reply:
Sorry I think I didn't phrase the comment clearly, it should be 2 events: `IN_PROGRESS` and `SUCCESS`. I have updated the comment!